mlir/lib/Dialect/Rock/IR/RockDialect.cpp

```diff
@@ -443,19 +451,23 @@ ConvolutionDims ConvolutionDims::fromOp(Operation *op) {
       ho = outputShape[i];
     } else if (outputAttr.getValue() == "wo") {
       wo = outputShape[i];
+    } else if (outputAttr.getValue() == "0o") {
+      ho = outputShape[i];
+    } else if (outputAttr.getValue() == "1o") {
+      wo = outputShape[i];
     }
   }
 
-  return ConvolutionDims(y, x, ho, wo, hi, wi, k, c, n, g);
+  return ConvolutionDims({y, x}, {ho, wo}, {hi, wi}, k, c, n, g);
```

Comment:
Because ConvolutionDims has arrays for those dimensions, we can aggregate them in the constructor call here.

The logic here will need brushing up to extend to 3-D.